x-pack/plugins/stack_connectors/common/sentinelone/schema.ts

```diff
@@ -388,39 +406,7 @@ export const SentinelOneGetRemoteScriptStatusResponseSchema = schema.object({
     nextCursor: schema.nullable(schema.string()),
   }),
   errors: schema.nullable(schema.arrayOf(schema.object({ type: schema.string() }))),
-  data: schema.arrayOf(
-    schema.object(
-      {
-        agentIsDecommissioned: schema.nullable(schema.boolean()),
-        agentComputerName: schema.nullable(schema.string()),
-        status: schema.nullable(schema.string()),
-        groupName: schema.nullable(schema.string()),
-        initiatedById: schema.nullable(schema.string()),
-        parentTaskId: schema.nullable(schema.string()),
-        updatedAt: schema.nullable(schema.string()),
-        createdAt: schema.nullable(schema.string()),
-        agentIsActive: schema.nullable(schema.boolean()),
-        agentOsType: schema.nullable(schema.string()),
-        agentMachineType: schema.nullable(schema.string()),
-        id: schema.nullable(schema.string()),
-        siteName: schema.nullable(schema.string()),
-        detailedStatus: schema.nullable(schema.string()),
-        siteId: schema.nullable(schema.string()),
-        scriptResultsSignature: schema.nullable(schema.nullable(schema.string())),
-        initiatedBy: schema.nullable(schema.string()),
-        accountName: schema.nullable(schema.string()),
-        groupId: schema.nullable(schema.string()),
-        agentUuid: schema.nullable(schema.string()),
-        accountId: schema.nullable(schema.string()),
-        type: schema.nullable(schema.string()),
-        scriptResultsPath: schema.nullable(schema.string()),
-        scriptResultsBucket: schema.nullable(schema.string()),
-        description: schema.nullable(schema.string()),
-        agentId: schema.nullable(schema.string()),
-      },
-      { unknowns: 'allow' }
-    )
-  ),
+  data: schema.arrayOf(schema.mapOf(schema.string(), schema.any())),
```

Comment:
Why change to this less restrictive schema?

Reply:
We don't want the external system API responses to be validated as tht could cause errors in Kibana that would otherwise be considered success in SentinelOne. 
I have a thread where I discussed this with the ResponseOps team (in their team channel I think) as we are looking to at some point do this for the other APIs we call.